docs/upgrading/from_seat_3_0.md

```diff
@@ -0,0 +1,187 @@
+# SeAT 3.0 to 4.0
+
+The upgrade path from SeAT 3.0 to SeAT 4.0 requires some manual work. This is mainly due to a large package revamp that
+have been made in SeAT 4.0.
+
+!!! info
+
+    Major changes have been applied in the way packages are working together, especially regarding eveapi, web and services ones.
+    Also, the way jobs are queued and queues themselve have been altered to reduce resources thumbrint and improve fluency.
+    
+    Before starting upgrade, pay attention on plugins you may use and ensure they are compatible with the new generation.
+    If it should not be the case, please come to us on Slack - so we will attempt to assist or redirect you to proper person.
+
+!!! warning
+
+    Before starting to do anything, be sure you read and understand this whole page content.
+
+    Also, remember to do at least a complete backup of your current database that you pick out of server on which SeAT
```

Comment:
Not sure what you mean by `pick out of server` do you mean copy the database to a local machine?

Reply:
Yes, or somewhere else than the server on which you'll perform the upgrade.
This is mostly a safety principle.